### PR TITLE
Display and UpperHex + formatting

### PR DESCRIPTION
**Pull request by simonmasson**
## Description

Change the `Display` and `UpperHex` functions for `BigIntegrer` in order to solve the issue of [this commit](https://github.com/arkworks-rs/algebra/commit/5f1a33181f08e862147f9836a69629a958ddff96).

I also ran `cargo fmt`.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (master)
- [ ] Linked to GitHub issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests
- [ ] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer

### CODE REVIEW

**Comment by simonmasson** [comment]
@Pratyush and @mmagician , can you review this PR ? :-)

File: ff/src/biginteger/mod.rs [hunk 1]
```diff
@@ -513,21 +514,15 @@ impl<const N: usize> FromBytes for BigInt<N> {
     }
 }
 
-impl<const N: usize> UpperHex for BigInt<N> {
```
Comment (mmagician) [inline ff/src/biginteger/mod.rs:516]: Wasn't that meant to stay?

**Review by mmagician — CHANGES_REQUESTED** [review]
I think your formatting might have been rum with nightly - since the CI job uses stable rust, they will likely mismatch.
Regardless, for readability it's probably easier to split it out into separate bugfix and formatting PRs.

File: ff/src/biginteger/mod.rs (hunk 1 above)
Comment (simonmasson) [reply ff/src/biginteger/mod.rs:516]: sure! I will seperate it in two PR with BigInt<N>... my bad (this time :-D)

**Pull request closed without merging**